Header header

geometry_msgs/Quaternion q_ItoG
geometry_msgs/Vector3 q_ItoG_eul_deg
geometry_msgs/Vector3 b_g_deg
geometry_msgs/Vector3 v_IinG
geometry_msgs/Vector3 b_a
geometry_msgs/Vector3 p_IinG
float32 b_alt
float32 t_d
geometry_msgs/Vector3 p_RinI
geometry_msgs/Quaternion q_RtoI
geometry_msgs/Vector3 q_RtoI_eul_deg